test/e2e/case2_config_deployment_test.go

```diff
@@ -68,7 +69,33 @@ var _ = Describe("Test config-policy-controller deployment", func() {
 				return getAddonStatus(addon)
 			}, 240, 1).Should(Equal(true))
 
-			By(logPrefix + "showing the config-policy-controller managedclusteraddon as available")
+			By(logPrefix + "annotating the managedclusteraddon with the " + loggingLevelAnnotation + " annotation")
```

Comment:
Nit: I would prefer these in a separate `It` test. Right now, if the annotation isn't applied it rolls up to the "should create the default ... deployment on the managed cluster" test.

In a separate test, it would also be nice to verify that the container becomes ready and doesn't crash immediately - that would verify that the provided values are valid.

But it's not necessary for me to feel ok with merging the PR.

Reply:
That's a good point. It does check to ensure a new pod is created and then running, but that test is getting a little bit long. I'll split it up.